pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -297,6 +305,94 @@ func (r *ReconcileStorageCluster) newStorageClasses(initData *ocsv1.StorageClust
 	return ret, nil
 }
 
+// newRBDSnapshotClass returns a new RBD snapshotter backed SnapshotClass
+func (r *ReconcileStorageCluster) newRBDSnapshotClass(initData *ocsv1.StorageCluster) *snapapi.VolumeSnapshotClass {
```

Comment:
```suggestion
func newRBDSnapshotClass(initData *ocsv1.StorageCluster) *snapapi.VolumeSnapshotClass {
```
since you're not accessing or mutating `ReconcileStorageCluster`